src/components/Auth/PublicRoute.js

```diff
@@ -8,7 +8,7 @@ const PublicRoute = ({ component: Component, ...rest }) => (
       !localStorage.getItem('token') ? (
         <Component {...props} />
       ) : (
-        <Redirect to='/dashboard' />
+        <Redirect to='/' />
```

Comment:
@kip-guile This component is preventing people that have logged in from going to register/login. It should redirect to their dashboard